libr/core/cmd.c

```diff
@@ -996,8 +997,55 @@ static int cmd_kuery(void *data, const char *input) {
 	const char *sp, *p = "[sdb]> ";
 	const int buflen = sizeof (buf) - 1;
 	Sdb *s = core->sdb;
+	
+	char *cur_pos, *cur_cmd, *next_cmd;
+	char *temp_pos, *temp_cmd, *temp_storage;
 
 	switch (input[0]) {
+
+	case 'j':
+		out = sdb_querys (s, NULL, 0, "anal/**");
+
+		r_cons_printf ("{\"anal\":{");
+
+		while (strlen (out) != 0) {
```

Comment:
use *out instead of strlen, you dont want to  know the length of the string, just if its empty or not